APE9.rst

```diff
@@ -0,0 +1,142 @@
+A General Timeseries Class for Astropy
+-------------------------------------
+
+author: Stuart Mumford
+
+date-created: 2016 March 24
+
+date-last-revised: 2017 November 10
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+The goal of a timeseries object in astropy is to provide a core set of
+functionality for representing data where time is the physical event list. This
+includes both continuous values binned in time or event list type data.
+
+The motivation for creation of this class is to support representation and
+analysis of astrophysical timeseries data, including support for the Units, Time
+and Coordinates packages in Astropy. This object should also be compatible with
+other packages in the wider scientific Python ecosystem, and try to minimise
+overlap in functionality with these packages.
+
+This document proposes an implementation based on Table, with extra constraints
+and specific functionality dedicated to timeseries type data.
+
+Detailed description
+--------------------
+
+Many different areas of astrophysics have to deal with 1D timeseries data, i.e.
+either sampling a continuous variable at fixed times or counting some events
+binned into time windows. These types of applications require some basic
+functionality:
+
+#. Extending timeseries with extra rows
+#. Concatenating multiple timeseries objects
+#. Sorting
+#. Slicing / selecting time ranges (indexing)
+#. Re-binning and re-sampling timeseries
+#. Interpolating to different time stamps.
+#. Masking
+#. Support for subtraction and addition (e.g. background).
```

Comment:
"Support for subtraction and addition" - is this just the same as being able to easily manipulate underlying data?